### PR TITLE
add type info described in issue #5236

### PR DESCRIPTION
**Pull request by burness**
A pr add the type info mentioned in [https://github.com/tensorflow/tensorflow/issues/5236](https://github.com/tensorflow/tensorflow/issues/5236)

### CODE REVIEW

**Comment by burness** [comment]
I signed it!

File: tensorflow/core/common_runtime/simple_placer.cc [hunk 1]
```diff
@@ -814,7 +814,8 @@ void SimplePlacer::AssignAndLog(const string& assigned_device,
   node->set_assigned_device_name(assigned_device);
   // Log placement if log_device_placement is set.
   if (options_ && options_->config.log_device_placement()) {
-    printf("%s: %s\n", node->name().c_str(),
+    printf("%s: (%s): %s\n", node->name().c_str(),
+           node->type_string().c_str(),
```
Comment (burness) [inline tensorflow/core/common_runtime/simple_placer.cc]: Yes， I will do it

**Comment by burness** [comment]
@vrv  Hi,  I have add this type info to the LOG message. Please review

File: tensorflow/core/common_runtime/simple_placer.cc [hunk 2]
```diff
@@ -815,9 +815,10 @@ void SimplePlacer::AssignAndLog(const string& assigned_device,
...
            node->assigned_device_name().c_str());
-    LOG(INFO) << node->name() << ": " << node->assigned_device_name();
+    LOG(INFO) << node->name() << ": " <<"("<<node->type_string()<<")"<< node->assigned_device_name();
```
Comment (vrv) [inline tensorflow/core/common_runtime/simple_placer.cc]: Please add spaces between the << and the quoted strings.

Comment (burness) [inline tensorflow/core/common_runtime/simple_placer.cc]: I have fix it

**Comment by vrv** [comment]
Nevermind, I did it for you (apparently GitHub allows that).

@tensorflow-jenkins test this please

**Comment by vrv** [comment]
This can't break cmake, so assuming infra failure.

**Review by vrv — APPROVED** [review]

**Pull request merged**